### DISCUSSION TITLE
url sceme registered in info.plist but mobile authentication gives error

### DISCUSSION THREAD

**Discussion by uday-itsok** [firebase/flutterfire#11015] [Q&A]
I have setup firebase authentication and it was working well earlier.
But now on release for ios it gives this error
`verification failed:...Please register custom URL scheme 'app-......' in the app's Info.plist file.`

I have already added url to the info.plist from xcode.
This is how info.plist looks like
```
...
<key>CFBundleURLTypes</key>
	<array>
		<dict>
			<key>CFBundleTypeRole</key>
			<string>Editor</string>
			<key>CFBundleURLSchemes</key>
			<array>
				<string>app-....</string>
			</array>
		</dict>
	</array>
....................

```

**Comment by yishayhaz** [comment]
Should be with the `REVERSED_CLIENT_ID` from `GoogleService-Info.plist`
